python/pyspark/testing/connectutils.py

```diff
@@ -76,6 +76,7 @@ class MockRemoteSession:
     def __init__(self):
         self.hooks = {}
         self.session_id = str(uuid.uuid4())
+        self.is_stopped = True
```

Comment:
this change is needed to make `test_connect_plan` with `MockRemoteSession` pass, otherwise

https://github.com/zhengruifeng/spark/actions/runs/5946008704/job/16125973924